common.py

```diff
@@ -82,7 +83,7 @@ def ipa_to_entries(input: str) -> List[IPRouteAddressEntry]:
 
         master = e["master"] if "master" in e else None
 
-        ret.append(IPRouteAddressEntry(e["ifindex"], e["ifname"], e["flags"], master, addr_infos))
+        ret.append(IPRouteAddressEntry(e["ifindex"], e["ifname"], e["flags"], master, e["address"], addr_infos))
```

Comment:
What is the purpose of this change?

Reply:
Aahh disregard I see now, we are using this field to determine the eth address of the vir bridge

Reply:
Yeah, I know it's confusing but it's actually called "address" in the json result.  So I kept it as is.